client/lib/plans/plans-list.js

```diff
@@ -357,6 +369,98 @@ const getPlanBusinessDetails = () => ( {
 	getInferiorHiddenFeatures: () => [],
 } );
 
+const getPlanJetpackSecurityDailyDetails = () => ( {
+	group: constants.GROUP_JETPACK,
+	type: constants.TYPE_SECURITY,
+	getTitle: () =>
+		translate( 'Jetpack Security {{em}}Daily{{/em}}', { components: { em: <em /> } } ),
+	getAudience: () => translate(),
+	availableFor: ( plan ) =>
+		[ constants.PLAN_JETPACK_FREE, ...constants.JETPACK_LEGACY_PLANS ].includes( plan ),
+	getDescription: () =>
+		translate(
+			'Enjoy the peace of mind of complete site protection. ' +
+				'Great for brochure sites, restaurants, blogs, and resume sites.'
+		),
+	getTagline: () => translate( 'Best for sites with occasional updates' ),
+	getPlanCompareFeatures: () => [],
+	getSignupFeatures: () => [],
+	getHiddenFeatures: () => [
+		constants.FEATURE_JETPACK_BACKUP_DAILY,
+		constants.FEATURE_JETPACK_BACKUP_DAILY_MONTHLY,
+		constants.FEATURE_JETPACK_SCAN_DAILY,
+		constants.FEATURE_JETPACK_SCAN_DAILY_MONTHLY,
+		constants.FEATURE_JETPACK_ANTI_SPAM,
+		constants.FEATURE_JETPACK_ANTI_SPAM_MONTHLY,
+	],
+	getInferiorHiddenFeatures: () => [],
+} );
+
+const getPlanJetpackSecurityRealtimeDetails = () => ( {
+	group: constants.GROUP_JETPACK,
+	type: constants.TYPE_SECURITY,
+	getTitle: () =>
+		translate( 'Jetpack Security {{em}}Real-time{{/em}}', { components: { em: <em /> } } ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **35** times:
`translate( 'Jetpack Backup {{em}}Real-Time{{/em}}' )` *ES Score: 7*